webapi/src/main/scala/org/knora/webapi/responders/v2/ValuesResponderV2.scala

```diff
@@ -554,7 +554,7 @@ class ValuesResponderV2 extends Responder {
             insertSparql = insertSparql,
             unverifiedValue = UnverifiedValueV2(
                 newValueIri = newValueIri,
-                value = valueToCreate.valueContent,
+                valueContent = valueToCreate.valueContent.unescape,
```

Comment:
The `UnverifiedValueV2` objects returned by `ValuesResponderV2` needed to be unescaped.